### ISSUE TITLE
Use Case: Provide feedback to a user as they manipulate the map

### ISSUE THREAD

**Issue by NickFitz** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#80]
This issue is for discussion of [the use case “Provide feedback to a user as they manipulate the map”](https://maps4html.github.io/HTML-Map-Element-UseCases-Requirements/#provide-feedback-user-they-manipulate-map), its examples & list of required capabilities.

**Comment by prushforth** [comment]
Like what kind of feedback, for example?

**Comment by AmeliaBR** [comment]
Maybe a clearer wording for this (authoring API) use case would be "Update web page content in response to user-initiated changes of the map view" ?

Or maybe we can split this up into multiple cases, one for reacting to pan & zoom, and another for reacting to layer selection/visibility changes.

**Comment by Malvoz** [comment]
Such feedback could also mean aural feedback, or have the device start vibrating as a person with such needs start closing in on her destination, for example.

I think, as an authoring capability it'd be a best practice for developers to be wary and provide feedback that is easy to understand/read: https://www.w3.org/WAI/RD/wiki/Easy_to_Read

**Comment by AmeliaBR** [comment]
> Such feedback could also mean aural feedback, or have the device (or otherwise a connected one) start vibrating

Good point. We can keep the definition open enough to any kind of change or action that website scripts can trigger in response to events.

> as a person with such needs start closing in on her destination, for example.

But this is starting to slide into a different use case: reacting to the user's physical geo-location, not just to their interaction with the map.

**Comment by prushforth** [comment]
The users physical location is important ,obviously, but not the only way to input a location.  For example, if a map was made static, so that dragging a finger over it didn't immediately pan it, the UA could vibrate or speak or otherwise communicate the feature being touched, potentially with a view to generating next steps, directions or other info about it. A haptic map, if such a thing exists.
